scripts/common

```diff
@@ -4,24 +4,35 @@ log (){
 export -f log
 
 bootstrap(){
-	local BOOTSTRAP_CMD=debootstrap
 	local BOOTSTRAP_ARGS=()
 
 	export http_proxy=${APT_PROXY}
 
-	if [ "$(dpkg --print-architecture)" !=  "arm64" ]; then
-		BOOTSTRAP_CMD=qemu-debootstrap
```

Comment:
So instead of using `qemu-bootstrap` we now use `bootstrap` but handle all the QEMU stuff manually?

Reply:
Exactly.